app/javascript/src/Types/index.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+
+export * from './Api'
```

Comment:
Didn't we remove relative path?

Reply:
Well this is an edge case. Index files are going to stay at the same dir forever and are meant to expose the files within. IMO it wouldn't make sense to use absolute paths for this particular case, however contradictory it seemed to be.

Reply:
So we can't use `export * from 'Api'` ?

I am really lost :)

Reply:
Absolute paths are _relative_ to `porta/app/javascript/src` so `export * from 'Api'` would be inexistent.